### ISSUE TITLE
Locast is suspending operations effective immediately.

### ISSUE THREAD

**Issue by mbtfeedback** [wouterdebie/locast2tuner#103]
Couldn't find a better place to post this, but due to a lawsuit, they're done. They sent out an email this morning and there is a message on their website.

**Comment by Thunder7ga** [comment]
Just saw this too...dang.  This sucks.

**Comment by Croq360** [comment]
Yeah, interesting how they have done this so quickly.  I thought there was still a trial coming up (which they were trying to avoid by asking for summary judgment in their favor).  I hope they figure out a way to restore it.

**Comment by rtm2017** [comment]
Now to look for alternatives, so far I haven't been very successful with anything other than pirated IPTV providers. Thanks for providing this tool though, it was glorious.

**Comment by nate602271** [comment]
The best part was you can record and keep the files. Heck, I had all sorts of TV shows being recorded. Sad to see it go so soon. Just subscribed on the 28th of August.

**Comment by wouterdebie** [comment]
Thank you all for the support! What started as a simple python script for myself to watch the New Orleans Saints in Sundays, grew to become a Rust project with a complete CI/CD pipeline, builds for multiple operating systems and architectures and an actual community of people. In the process I've learned a new program language, learned a lot about Docker and GitHub actions and met some really awesome people.

I assume locast.org is gone and the future of this project is uncertain. Maybe I'll add other services. Let me know what y'all think. That said, thank you all for being an active member of this community! 

Lastly, I'd like to give a big shout-out to @Croq360, who abused the hell out of locast2dvr, which prompted me to redo everything in Rust and @eiddor for his amazing help with running everything in Docker!

**Comment by Thunder7ga** [comment]
Other services would be awesome.  Thank you for all you have done in developing this tool, it's been extremely valuable to my family.

**Comment by sumocomputers** [comment]
We live in the mountains where OTA reception is impossible. My wife likes Jeopardy and I am cheap, so Locast (and of course locast2tuner) was a great solution. It was good while it lasted. Now on to find the cheapest way to watch Jeopardy in the US...

**Comment by okc** [comment]
> ...My wife likes Jeopardy and I am cheap...Now on to find the cheapest way to watch Jeopardy in the US...
Interesting that you mention Jeopardy!  Jeopardy is apparently produced by Sony and distributed by CBS, but it is syndicated.  In my area it is on the CBS O/O station, so I can watch it live on the CBS app, but there is no way to watch it on demand, there is no way to watch prior seasons, etc.
